images/README.md

```diff
@@ -35,17 +35,15 @@
 
 8. Save, comment and upload changes to GitHub.
 
-10. Save, add, comment and upload changes to **GitHub**.
-
-11. Create a PR pointing to `develop-[name]` with title.
+9. Create a PR pointing to `develop-[name]` with title.
 
     > example: `skyline added` -> `[imageName] -> added`.
 
-12. Agregar a Edgar o Andrés como Reviewer
+10. Agregar a Edgar o Andrés como Reviewer
 
-13. Track the PR (updates, comments, changes).
+11. Track the PR (updates, comments, changes).
 
-14. Download at least 2 more images and repeat all the steps for each one.
```

Comment:
This changes are unrelated with the ticket because there are some other tickets fixing this